VTUL/dlp-access
# Explore collections page and Collection view page

**Adds collections list and collections view pages. This PR addresses LIBTD-1932 AND LIBTD-1933.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1932, https://webapps.es.vt.edu/jira/browse/LIBTD-1933) (:star:)

# What does this Pull Request do? (:star:)
Adds collections list and collections view pages.

# What's the changes? (:star:)
* Adds collections list page to display all collections with links to individual view pages.
* Adds route with optional page number and results limit parameters to support pagination
* Adds Collection view page to display details of individual collection. Includes links to any sub-collections and/or items that belong to the collection.

# How should this be tested?
* Check that collections list page works as expected with links to individual collections
* Check that collections view page works as expected with links to sub-collections and items

# Additional Notes:
* branch: `LIBTD-1932`

# Interested parties
@yinlinchen 

(:star:) Required fields
